app.js

```diff
@@ -11,10 +11,57 @@ var express = require('express'),
 Habitat.load();
 
 var app = express(),
-  env = new Habitat();
+  env = new Habitat(),
+  OPTIMIZELY_ID = env.get('OPTIMIZELY_ID'),
+  OptimizelyClient = require('optimizely-node-client'),
+  optimizely = new OptimizelyClient(env.get('OPTIMIZELY_TOKEN'));
 
 app.set('trust proxy', true);
 
+app.use( '/optimizely-experiments.js', function( req, res ) {
```

Comment:
This creates a data file for optimizely data that the client can use.

We could use this to inject react into the test code scope.